service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -70,22 +79,105 @@ private void run3037(CaseDetails caseDetails) {
+                    caseData.getFamilyManCaseNumber(), SECOND_BUNDLE_ID, secondBundle.getId()));
+        }
 
+        Map<String, Object> data = caseDetails.getData();
         data.put("additionalApplicationsBundle", additionalApplicationsBundle);
     }
+
+    private List<Element<SupportingEvidenceBundle>> buildSupportingEvidenceBundle() {
+        SupportingEvidenceBundle firstBundle = SupportingEvidenceBundle.builder()
+            .name("Position Statement for C2")
+            .document(DocumentReference.builder()
+                .url("http://dm-store-prod.service.core-compute-prod.internal/documents/d448738d-51fe-439e-9a0b-5ecc0bb378b6")
+                .binaryUrl("http://dm-store-prod.service.core-compute-prod.internal/documents/d448738d-51fe-439e-9a0b-5ecc0bb378b6/binary")
+                .filename("Position Statement for C2.docx")
+                .build())
+            .uploadedBy("HMCTS")
+            .dateTimeUploaded(LocalDateTime.of(2021, 05, 24, 10, 23, 32))
+            .build();
+
+        SupportingEvidenceBundle secondBundle = SupportingEvidenceBundle.builder()
+            .name("Draft LOI")
+            .document(DocumentReference.builder()
+                .url("http://dm-store-prod.service.core-compute-prod.internal/documents/d8357dbc-e3bd-464f-8edb-aac0fe1c58c2")
+                .binaryUrl("http://dm-store-prod.service.core-compute-prod.internal/documents/d8357dbc-e3bd-464f-8edb-aac0fe1c58c2/binary")
+                .filename("AMO0030002 Draft LOI.docx")
+                .build())
+            .uploadedBy("HMCTS")
+            .dateTimeUploaded(LocalDateTime.of(2021, 05, 24, 10, 23, 32))
+            .build();
+
+        SupportingEvidenceBundle thirdBundle = SupportingEvidenceBundle.builder()
+            .name("CV")
+            .document(DocumentReference.builder()
+                .url("http://dm-store-prod.service.core-compute-prod.internal/documents/6b6b5071-e097-4074-b317-b00dc3cfa89c")
+                .binaryUrl("http://dm-store-prod.service.core-compute-prod.internal/documents/6b6b5071-e097-4074-b317-b00dc3cfa89c/binary")
+                .filename("AMO0030002 Medico-legal_CV-May2021.doc")
+                .build())
+            .uploadedBy("HMCTS")
+            .dateTimeUploaded(LocalDateTime.of(2021, 05, 24, 10, 23, 32))
+            .build();
+
+        return wrapElements(firstBundle, secondBundle, thirdBundle);
+    }
+
+    private void swapC2WithSupportingDocument(AdditionalApplicationsBundle application,
+              List<Element<AdditionalApplicationsBundle>> additionalApplicationsBundle) {
+        C2DocumentBundle c2DocumentBundle = application.getC2DocumentBundle();
+        Optional<Element<SupportingEvidenceBundle>> supportingEvidenceBundle = c2DocumentBundle
+            .getSupportingEvidenceBundle()
+            .stream()
+            .filter(bundle -> bundle.getId().equals(SUPPORTING_EVIDENCE_ID))
+            .findFirst();
+
+        if (supportingEvidenceBundle.isPresent()) {
```

Comment:
Is there a possibility that it won't be present?

Reply:
no guess not - will remove :)